packages/components/src/Text/Text.mdx

```diff
@@ -69,3 +58,25 @@ only be used right after a page title.
     Drag to rearrange the order that fields show up in Jobber
   </Text>
 </Playground>
+
+## Sizes
+
+## Intro
+
+`Text` can be used with different sizes.
+
+Typically `large` is to be used as an intro text that gives more information
+about the page title and can only be used right after a page title.
+
+<Playground>
+  <Text size="large">
+    Attract new customers by asking happy clients for a Facebook recommendation
+  </Text>
+</Playground>
+
+The `small` size is intended to be used as helper text on things like
+checkboxes.
```

Comment:
suggest expanding this a bit, as I expect this to be a big styling component. 
`checkboxes.` -> `checkboxes and other form inputs, or smaller context help for other UI or content.`

Reply:
This is where Typescript will help I think. The default size of `Typography` is not `small`. So any invalid prop there would render the default size of typography. 

In real world usage of the examples above (outside of Docz) typescript would catch this and complain about an invalid prop value.

![image](https://user-images.githubusercontent.com/46460840/99568005-1d612c80-298c-11eb-8773-dd7f83bbd6c2.png)

![image](https://user-images.githubusercontent.com/46460840/99568059-3073fc80-298c-11eb-8579-8429fef1eccb.png)

Reply:
Yes, you are totally right! Ok, this is fine. Its weird (can we fix it anyway?) but its not standard behavior.

Reply:
Talked a bit more, and it is not only "nonstandard" it is invalid TypeScript. So, no need to write code that replaces that!